pkg/harvester/l10n/en-us.yaml

```diff
@@ -10,10 +10,10 @@ generic:
 nav:
   group:
     networks: Networks
-    backupAndSnapshot: Backup & Snapshot
+    backupAndSnapshot: Backup and Snapshots
     Monitoring: Monitoring
     Logging: Logging
-    'Monitoring & Logging': Monitoring & Logging
+    'Monitoring and Logging': Monitoring and Logging
```

Comment:
Since changing the translation key, remember to update the [caller](https://github.com/harvester/dashboard/blob/b97bf0600b4da2bb0d11fd70d7c0af0cf01789cb/pkg/harvester/config/harvester.js#L40-L41).

Reply:
Of course, thanks! I'm still finalising /assets/ - then I'll jump on the caller(s).